src/com/TTT/MiniMax.java

```diff
@@ -0,0 +1,56 @@
+package com.TTT;
+
+public class MiniMax {
+
+    private String currentPlayerSymbol;
+
+    public int nextMove(Board board) {
+        int bestScore = -20;
```

Comment:
Might be nice to make these constants, so that you can refer to them in `getScoreForMove` as well.